src/native-types.ts

```diff
@@ -1673,6 +1701,16 @@ HKCategorySampleRaw<TCategory>,
 'device' | 'endDate' | 'startDate' | 'uuid'
 >;
 
+export interface HKWorkoutEvent {
+  readonly type: string,
```

Comment:
Is this really a string - it looks like an int in the docs?

To follow the pattern used across the library an enum like this would probably be a good fit:
```TypeScript
enum HKWorkoutEventType {
  pause = 1,
  resume = 2,
  // ...
}
```

Reply:
Thanks good idea, updated